src/rsa.rs

```diff
@@ -295,7 +296,16 @@ impl RSAKeyPair {
                 try!(bssl::map_result(unsafe {
                     rsa_new_end(rsa.as_mut())
                 }));
-                Ok(RSAKeyPair { rsa: rsa })
+                let blinding = unsafe { BN_BLINDING_new() };
+                if blinding.is_null() {
+                    Err(())
+                }
+                else {
```

Comment:
Ha, that was actually what I first wrote. I'll go with the early return, with the `} else` style noted in case it pops up again.